Web/starter_project_vanguard/next.config.js

```diff
@@ -1,6 +1,17 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  images: {
```

Comment:
same comment that i had on your other PR here to

Reply:
Done